File: solutions/JavaScript/1431.KidsWithTheGreatestNumberOfCandies.js
var kidsWithCandies = function (candies, extraCandies) {
    const MAX_CANDIES = Math.max(...candies);
    let enoughCandies = [];
    for (candy of candies) {
        enoughCandies.push(candy + extraCandies >= MAX_CANDIES);
    }
    return enoughCandies;
};

File: solutions/Python/1431.KidsWithTheGreatestNumberOfCandies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        enoughCandies = []
        MAX_CANDIES = max(candies)
        for candy in candies:
            if candy + extraCandies >= MAX_CANDIES:
                enoughCandies.append(True)
            else:
                enoughCandies.append(False)
        return enoughCandies